neutron/assetlist.json

```diff
@@ -313,6 +313,55 @@
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/dydx/images/dydx.png",
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/dydx/images/dydx.svg"
       }
+    },
+    {
+      "description": "ASTRO on Neutron",
+      "denom_units": [
+        {
+          "denom": "ibc/5751B8BCDA688FD0A8EC0B292EEF1CDEAB4B766B63EC632778B196D317C40C3A",
+          "exponent": 0,
+          "aliases": [
+            "uastro"
+          ]
+        },
+        {
+          "denom": "astro",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/5751B8BCDA688FD0A8EC0B292EEF1CDEAB4B766B63EC632778B196D317C40C3A",
+      "name": "Astroport token",
+      "display": "astro",
+      "symbol": "ASTRO",
+      "traces": [
+        {
+          "type": "ibc",
```

Comment:
For cw20 assets, please use trace type 'ibc-cw20', which adds a couple more required properties (i.e., 'port')